src/app/CommandHandler.cpp

```diff
@@ -550,6 +553,18 @@ CHIP_ERROR CommandHandler::FinishStatus()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CommandHandler::RollbackResponse()
+{
+    VerifyOrReturnError(mState == State::Idle || mState == State::AddedCommand || mState == State::AddingCommand,
```

Comment:
I feel like we should have a new state for "checkpointed our builder, but did not complete PrepareStatus/PrepareCommand" and check for that state, not Idle, here.  Followup is OK for this change.

Reply:
I think we don;t need such state, but we should change the order of checking and setting states in PrepareStatus